pom.xml

```diff
@@ -57,11 +57,11 @@
     <version.org.eclipse.jdt.core.compiler>4.6.1</version.org.eclipse.jdt.core.compiler>
     <version.com.github.javaparser>3.13.10</version.com.github.javaparser>
     <version.com.fasterxml.jackson>2.9.9</version.com.fasterxml.jackson>
-    <version.kubernetes-client>4.3.1</version.kubernetes-client>
+    <version.kubernetes-client>4.4.2</version.kubernetes-client>
     <version.okhttp>3.12.0</version.okhttp>
```

Comment:
Could you please check if this version is compatible with the Kubernetes Client also?

Reply:
Kubernetes Client has also been tested with it.
In fact this update is a prerequisite to  KOGITO-120 which implements some integration tests on kogito-cloud libraries

Reply:
Kubernetes Client uses this specific version: https://github.com/fabric8io/kubernetes-client/blob/v4.4.2/pom.xml